pkg/service/v0/settings.go

```diff
@@ -121,3 +133,136 @@ func generateBundleProfileRequest() settings.SaveBundleRequest {
 		},
 	}
 }
+
+func generateProfilePermissionsRequests() []settings.AddSettingToBundleRequest {
+	// TODO: we don't want to set up permissions for settings manually in the future. Instead each setting should come with
+	// a set of default permissions for the default roles (guest, user, admin).
+	return []settings.AddSettingToBundleRequest{
+		{
+			BundleId: ssvc.BundleUUIDRoleAdmin,
+			Setting: &settings.Setting{
+				Id:          "7d81f103-0488-4853-bce5-98dcce36d649",
+				Name:        "language-create",
+				DisplayName: "Permission to set the language",
```

Comment:
do we have a way to translate those display names yet ?

Reply:
Not in the services. The displayname is shown in the UI and transifex is used on it, but since this is dynamically rendered, I'm sure that transifex is not able to pick those strings, yet. Needs investigation or some help from the transifex gods.

Reply:
https://github.com/owncloud/ocis-settings/issues/51